apps/src/templates/progress/ProgressBubble.jsx

```diff
@@ -176,6 +178,7 @@ class ProgressBubble extends React.Component {
         tooltipId={tooltipId}
         icon={levelIcon}
         text={tooltipText}
+        includeAssessmentIcon={levelIsAssessment}
```

Comment:
nit: since we don't consume the value of `levelIsAssessment` anywhere else, we can call `isLevelAssessment(level)` like this:

```suggestion
        includeAssessmentIcon={isLevelAssessment(level)}
```

and delete the assignment to `levelIsAssessment` on line 127

Reply:
So I had that thought and then left it like this because I'm probably going to need this variable to do some of the other assessment ux work such as adding the little check-circle in the corner of the bubble.

Reply:
ah, understandable! makes sense to leave it as-is then